src/api/status/src/server.js

```diff
@@ -54,5 +69,39 @@ service.router.get(`${process.env.PATH_PREFIX || ''}/status`, (req, res) => {
     .catch((err) => res.status(500).json({ error: err.message }));
 });
 
+// Get Home page
+service.router.get(process.env.PATH_PREFIX || '/', async (req, res) => {
+  const [telescope, satellite, totalPost, totalFeeds] = await Promise.all([
+    getGitHubData('Seneca-CDOT', 'telescope'),
+    getGitHubData('Seneca-CDOT', 'satellite'),
+    getPostsCount(),
+    getFeedCount(),
+  ]);
+  let environment = {};
+  if (
```

Comment:
Flip your logic: check for the production URL do the `else` for everything else.

Reply:
Reversed